main.css

```diff
@@ -460,4 +425,39 @@ textarea {
   align-items: stretch;
   justify-content: center;
   color: #fff;
-}
\ No newline at end of file
+}
+
+@media (min-width: 768px) {
+  #flexbox-head-container {
+    background-image: url("img/mbl.svg");
+  }
+
+  .Flexbox-header {
+    height: 72px;
+  }
+
+  .NavToggle {
+    display: none;
+  }
+
+  .navigation_list {
+    list-style-type: none;
+    gap: 2em;
+  }
+
+  .navigation_head {
+    display: inline-block;
+  }
+
+  .section--headline {
+    margin: 0 auto;
+  }
+
+  .headline__title {
+    font-size: 48px;
+  }
+
+  .headline__description {
+    font-size: 20px;
+  }
+}
```

Comment:
As suggested by the previous reviewer, you are to meet all project requirements. kindly implement the following design changes for your contact form to enable you to have a better UI

1. please add the background images as shown in the Figma template given
2. kindly use the right margin annd padding for the input content 
3. Also, please remember to use the right styling for the button
4. lastly please fix the margin top for this section as it appears to be too much comapred to our figma template

your design :
![image](https://github.com/AsciencioAlex/Personal-Portfolio/assets/20537712/5d9063b5-0ecd-4af0-b9e2-92472b19d923)

Figma design :
![image](https://github.com/AsciencioAlex/Personal-Portfolio/assets/20537712/af072553-ee8b-4d7b-a0b1-71c57ae1da42)